nucypher/characters/control/interfaces.py

```diff
@@ -45,7 +45,9 @@ def wrapped(instance, request=None, *args, **kwargs) -> bytes:
         response_with_metadata = instance.serializer.build_response_metadata(response=response, duration=duration)
 
         # Emit
-        return instance.emitter(response=response_with_metadata)
+        emitted_response = instance.emitter(response=response_with_metadata)
+        emitted_response.headers["Content-Type"] = "application/json"
```

Comment:
At the moment, `emitted_response` may be an `int` of the number of bytes written.  Instead, set the headers in the `WebEmitter` class here: <https://github.com/nucypher/nucypher/blob/master/nucypher/characters/control/emitters.py#L140>

Reply:
Fixed. Also note, the response for `{{base_uri}}:{{enrico_port}}/encrypt_message` is still using `text/html` not sure where that comes from